src/models/user-status.model.ts

```diff
@@ -6,30 +6,29 @@ import { Platform, Status, Reason } from '../core';
 
 @Entity({ name: 'user_statuses' })
 export class UserStatus extends AuthenticatedRequest {
-  @PrimaryGeneratedColumn()
-  private id: number;
+  @PrimaryGeneratedColumn() private id: number;
 
-  @Column({ name: 'user_id', type: 'varchar', length: 30 })
+  @Column({ name: 'user_id', type: 'varchar', length: 30, nullable: false })
```

Comment:
You can add the `postId` also in this PR.

Reply:
Implemented with 581bf277cfcd6fa39ab547044d2b00ccc8a109f1